app/views/users/profile_edit/_donations_by_causes.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render "users/profile_sections/profile_section", title: "DONATIONS BY CAUSES"  do %>
+  <div class="donations-bars">
```

Comment:
we could just rename this to just `donations`. You also don't have any styling for this class so we shouldn't need it

Reply:
I have some styling for it. Look in `users/profile_edit/donations_by_causes.scss`. I don't `donations` would be a good name for it because it isn't a group of donations. Each item is not a donation. I think that might be a bit misleading.

Reply:
So for CSS, you have to be cognitive on what we are **seeing**, as CSS is applied to what we actually see. From the view, it literally says "Donation By Causes", which can fairly point out that this div element is being applied with a `donations` class tag.

However, looking further into it, something like `donation-distribution` might make more sense.

Reply:
I'll probably go with `donation-distribution`. I think that sounds good.